patient_portal/lib/appsql.class.php

```diff
@@ -260,11 +273,18 @@ public function escapeHtml( $string ){
      * @param Date format set in GLOBALS
      * @return Date format in PHP
      */
-    public function dateFormat( $format ){
-        if( $format == "0" ) $date_format = 'yyyy/mm/dd';
-        else if( $format == 1 ) $date_format = 'mm/dd/yyyy';
-        else if( $format == 2 ) $date_format = 'dd/mm/yyyy';
-        else $date_format = $format;
+    public function dateFormat($format)
```

Comment:
We have date formats defined in globals.

Reply:
This is old code that was reused for the code base.